### PR TITLE
fix: apply grab cursor to assigned categories dimension item

### PR DESCRIPTION
**Pull request by HendrikThePendric**
Implements [DHIS2-15140](https://dhis2.atlassian.net/browse/DHIS2-15140)

---

### Key features

1. Updates cursor style for assigned categories dimension item

---

### Screenshots

| When/where | Style rule | Screenshot |
| ------------- | ------------- | ------------- |
| Sidebar unselected  | `cursor: grab;`  | sidebar_unselected | 
| Sidebar selected  | `cursor: default;`  | sidebar_selected | 
| Layout  | `cursor: grab;`  | layout | 

[DHIS2-15140]: https://dhis2.atlassian.net/browse/DHIS2-15140?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/DimensionsPanel/DndDimensionItem.js [hunk 1]
```diff
@@ -41,11 +42,11 @@ export class DndDimensionItem extends Component {
...
                             {...provided.dragHandleProps}
-                            className={
-                                snapshot.isDragging
-                                    ? styles.dragging
-                                    : styles.notDragging
-                            }
+                            className={cx({
+                                [styles.dragging]: snapshot.isDragging,
+                                [styles.notDragging]: !snapshot.isDragging,
+                                [styles.assignedCategories]: id === 'co',
```
Comment (jenniferarnesen) [inline src/components/DimensionsPanel/DndDimensionItem.js:48]: Should "co" be put into a constant?

Comment (HendrikThePendric) [reply src/components/DimensionsPanel/DndDimensionItem.js:48]: Yeah, ideally it should. I looked for a suitable file, but couldn't fine one. Then I considered if it was worth creating a completely new file for a single 2-character constant that is only used in 2 places, and concluded it probably isn't.

But I'd be happy to move it somewhere else. Could you advise which file to use/create?

Comment (edoardo) [reply src/components/DimensionsPanel/DndDimensionItem.js:48]: Import `DIMENSION_ID_ASSIGNED_CATEGORIES` from `@dhis2/analytics`

Comment (HendrikThePendric) [reply src/components/DimensionsPanel/DndDimensionItem.js:48]: 🙏

**Review by janhenrikoverland — APPROVED** [review]

**Pull request merged**